build.gradle

```diff
@@ -69,6 +69,7 @@ project.description = 'Vividus'
 ext {
     versions = [
         allure:                 '2.13.5',
+        commonsCollections4:    '4.4',
```

Comment:
Used once dependencies should not be moved to versions because dependabot unable to update them